include/exadg/operators/variable_coefficients.h

```diff
@@ -23,168 +23,213 @@
 
 namespace ExaDG
 {
+/**
+ * This class serves as a manager of several table objects, which hold quadrature-point-level
+ * coefficient information for the different matrix-free loop types and access methods.
+ *
+ * The class provides functionality for cell loops, separate face loops, and cell-based face loops.
+ * For the separate face loops, coefficient information from the neighbor cell can also be stored
+ * and used.
```

Comment:
This sentence can now be removed.

Maybe replace "functionality" by "data structures" (or similar) in the sentence above, or is "functionality" intended here?

Reply:
I will replace "functionality" by "data structures", better than removing in my opinion.

Reply:
with removing I only mean the second sentence, because we skipped one boolean parameter.

Reply:
Oh yes, I see 👍